File: OOP/OOP_variable/set_get.py
class C(object):
    def __init__(self, v):  # 인스턴스 변수 self.value를 선언하여 v라는 생성자를 준다.
        self.value = v

    def show(self): # 인스턴스 변수 self.value를 출력한다.
        print(self.value)

    def getValue(self): # 인스턴스 변수의 값을 가져온다.
        return self.value

    def setValue(self, v): # 인스턴스 변수의 값을 설정한다.
        self.value = v

c1 = C(10)
print(c1.getValue())

c1.setValue(20) # 인스턴스의 변수의 값을 변경 한 결과.
print(c1.getValue())

File: OOP/OOP_variable/set_get.rb
class C
    def initialize(v)
        @value = v #인스턴스 변수 @value에 v라는 생성자를 준다.
    end

    def show() # 인스턴스 변수 @value의 값을 출력
        p @value
    end

    def getValue() # 인스턴스 변수 @value의 값을 가져온다.
        return @value
    end

    def setValue(v) # 인스턴스 변수 @value의 값을 설정한다.
        @value = v
    end
end

c1 = C.new(10)
p c1.getValue()

c1.setValue(20) # 인스턴스 변수 @value 값을 변경한 결과.
p c1.getValue()